languages/java/exercises/concept/strings/.docs/introduction.md

```diff
@@ -0,0 +1,7 @@
+A `string` in Java is an object that represents immutable text as a sequence of Unicode characters (letters, digits, punctuation, etc.). Double quotes are used to define a `string` instance:
+
+```java
+String fruit = "Apple";
+```
+
+Strings are manipulated by calling the string's methods. Once a string has been constructed, its value can never change. Any methods that appear to modify a string will actually return a new string.
```

Comment:
The example implementation used `String.format`. It might be good to reference that there are static `String` methods to use too.

By the way, I've just noticed that the C# track's README does not mention how to concatenate strings, so I should be adding that. Might be useful to add that here too.